assets/promptflow/evaluators/models/hate-unfairness-evaluator/spec.yaml

```diff
@@ -4,7 +4,9 @@ path: ./
 properties:
   is-promptflow: true
   is-evaluator: true
+  show-artifact: true
```

Comment:
for content safety ones, it shouldn't show the aritfact. please revert the change

Reply:
So the rules are:

- no artifact for content safety evaluators
- prompt template for prompt-based evaluators
- show the implementation file (e.g. _bleu.py) for all others

Is that correct?

Reply:
Yes!